whois-commons/src/main/java/net/ripe/db/whois/common/DataSourceConfigurations.java

```diff
@@ -59,24 +58,19 @@ public ComboPooledDataSource nrtmSlaveDataSource(@Value("${nrtm.slave.database.u
     private ComboPooledDataSource createDataSource(final String jdbcDriver, final String jdbcUrl, final String jdbcUser, final String jdbcPass) throws PropertyVetoException {
         final ComboPooledDataSource source = new ComboPooledDataSource();
 
-        source.setProperties(getSourceProperties(jdbcDriver));
```

Comment:
shouldn't be the same
setting properties on this way on another?
using setProperties or using setMinPoolSize...etc

Reply:
Ok, I see that you are doing this change in internal as well...
I don't understand why it is no working using `setProperties` 
I will close the issue but I am curious now